dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml

```diff
@@ -20,4 +20,9 @@
       <name>getClassLoader</name>  
    </permission>
 
+   <permission>  
+	  <class-name>java.lang.RuntimePermission</class-name>
+      <name>accessClassInPackage.com.sun.beans.editors</name>  
+   </permission>
+
```

Comment:
What changed that requires this permission?

Reply:
Could you also clean up the formatting of this class it's a bit messy?

Reply:
Opened for investigation: https://github.com/OpenLiberty/open-liberty/issues/12284